### DISCUSSION TITLE
Deep Work (Sept/Oct 2021)

### DISCUSSION THREAD

**Discussion by tomcudd** [Virtual-Coffee/virtualcoffee.io#316] [VC Book Club]
### Schedule

- Sept 1-7: Introduction
- Sept 8-14: Chapter 1
- Sept 15-21: Chapter 2
- Sept 22-28: Chapter 3
- Sept 29-Oct 5: Rule 1
- Oct 6-12: Rule 2
- Oct 13-19: Rule 3
- Oct 20-26: Rule 4
- Oct 27-31: Conclusion

I'll post the topic, conversation points, and questions for each week outlined above as I get through that part of the book. I will post links to the discussion in slack as well.

**Comment by tomcudd** [comment]
### Sept 1-7: Introduction

- Do you think deeper, careful thought can help you solve problems you are facing today?
- How might it be possible for us to prioritize or commit to intellectual effort needed to solve bigger problems?
- Many influential individuals in the last few decades of technology advances are working in contrast to knowledge workers today. Where in the spectrum of deep and shallow work do you think you fall today?
- Is your time to solve things only available in fragments or slivers today?
- Is shallow work expanding to fill up your working moments?

**Reply by tomcudd** [reply]
Generally speaking, one thing that stood out to me was the realization that much of my most successful efforts over the years have been when I dump all work but the most important or critical item and just focus on that.

**Reply by cmcrawford2** [reply]
The intro totally validated my experience in graduate school in the 2000s. I had left my job in the software industry to study the philosophy of A.N. Whitehead. The only institution in the Boston area that taught Whitehead was the Boston University School of Theology. They had a PhD program in Religion, Science and Philosophy that was really attractive to me. So I enrolled in their Master of Theological Studies program, but it was tough going and I gave up on the PhD after getting the MTS. As a full-time student, also a wife and mother, I had all kinds of obligations pulling at me at all times and never had enough time to read and absorb the difficult material that I wanted to learn.
Right now I am constantly fighting the pull of so-called "shallow" obligations. One doctor's appointment or grocery shopping trip pretty much splinters the rest of my day into little bits.
I am constantly trying to improve my time management methods to fight what a friend called "the momentum of ordinary events."

**Reply by jonathanyeong** [reply]
I'm totally onboard with this idea of Deep Work! I've gotten further on projects where I've had no distractions and was able to get into a flow state. Contrast to these days with being fully remote, and I feel like I'm answering slacks and having adhoc meetings way more. I'm closer to the shallow work spectrum these days. And I find it hard to carve out time to do deep work.

In terms of how to prioritize or commit effort to solve problems. From the intro, it seems like people he interviewed had long chunks of time where they could disconnect from the world. Benn holing himself into a room with just textbooks, Gates' week long retreats, Jung's Tower. And I know later in the book he'll go more into the mechanics of deep work. But wanted to float the idea that long periods of disconnectedness may not be feasible for everyone.

### A thought that I haven't quite finished but it might be interesting
**Feel Free to ignore!**
_What I find interesting is the way Cal talks about Deep Work. He mentions "professional activities", does that mean Deep Work is only during your day job? Can we extend deep work to activities outside of work like side projects and blog posts? And what does the balance of deep work vs shallow work look like at that point? Personally, I do more shallow work after 5pm, but..._

**Reply by tomcudd** [reply]
As I read through the intro, I also thought that there's a certain amount of privilege that comes with being able to just set aside a chunk of time to complete this type of deep level work.  It's sort of like eating right and exercising enough. It's difficult, but possible to prioritize if you happen to be in a stage of life that allows it. I have had times where that worked for me, and other times where it has been nearly impossible.

**Reply by cmcrawford2** [reply]
I think if I had had such a clear explanation of the concept of deep work when I was trying to study philosophy, then I would have been able to make some better choices. There's always something you can improve. I didn't have to be a full time student, for example. I didn't have to collect so many shiny objects (extra classes in interesting subjects). Now, same as then, I have a family who depend on me for things like making dinner and managing the finances. I've made time management a top priority since last year. The author's idea of deep work has really increased the relevance of that priority for me. So I've turned up the heat on some of my time management practices and I'm getting some hours of work in that feel "deep" even though I still have a lot of unrelated commitments.

**Comment by tomcudd** [comment]
### Sept 8-14: Chapter 1, Deep Work Is Valuable

- Does it seem attainable to become one of the groups of people excelling in today's digital workforce? (High skilled workers or superstars)
- Is it possible to improve these two core skills? (Quickly master difficult skills or produce at at elite level with quality and speed)
- Do you need to concentrate to learn? Do you need to practice deliberately?
- Do you struggle to find the time to focus or batch your difficult work into uninterrupted chunks of time?
- Do you plan on how you are going to get your work done? ("meta-planning" how you work instead of the specifics of what you work on)
- Do you see this equation work in your life? High quality work product = (time spent) x (intensity of focus)
- Do you feel "attention residue" when you get interrupted or switch tasks?
- Are you semi-distracted when you work? (Keeping email, slack, etc. running)

**Reply by cmcrawford2** [reply]
In general when I read a new book I give the author a lot of the benefit of the doubt. So for the first two questions, I do think it's possible, at least for me, to become a skilled worker. Not necessarily a superstar but why rule that out? Think positive! I also think it's possible to improve the two core skills. For me especially, I'm part way there because my background in numerical computing has made it easier to understand ML concepts.
I do struggle to find time to focus and to batch work. I have a lot of obligations pulling at me (making dinner for the family, housekeeping, commitments I've accumulated, hobbies, and a wide range of non-tech projects. Also learning new skills in my case is haphazard because there's no one purpose to what I'm learning. Some ML, some front end, some back end. So I have lots of classes ongoing at the same time and half a dozen programming projects.
I was looking at my smartphone way too much but then it broke. I still have email to distract me, and Slack, but I'm pretty good at putting it aside when I decide it's time to work. But because I haven't yet managed to batch large amounts of time to work on one thing over many days, I don't think I'm where I want to be yet.
Also I'm planning my day on paper in a narrow column, the way he says to do in a later chapter, and then writing down next to it what actually was done. If I get caught in email or Slack when I was planning to work, it's usually for 20 minutes to half an hour, and I make a note of it in the actual column. I would say right now it's about a once-a-day occurrence.

**Reply by cmcrawford2** [reply]
One thing I wanted to say about that chapter was that I wish he could have spent more than a sentence or two about how what he singles out as valuable is not the last word. I don't have the book in front of me right now, but he says these aren't the only ways people can end up on the plus side of the great restructuring. I know he gets to choose what he's going to write about, but I would have liked some attention given to other ways of creating value in the economy. I think it was someone on a previous Slack thread who pointed out to me that the ability to get people to work together is also important and valuable.

Also the whole premise of the great restructuring (or whatever he calls it, I might have that wrong) is something I am skeptical about. People have never really been able to predict the future. They extrapolate from the present, or they identify one thing that's changing and assume the rest is going to stay the same, and that's almost never how the future ends up working out. But I don't think that matters much. I definitely would like to be able to do work that I'm proud of, and to not be distracted seems like a good idea, whether or not the world ends up looking like he thinks it will.

**Reply by jonathanyeong** [reply]
Lots of discussion points! I agree with @cmcrawford2, I wish he would delve a little deeper into the other groups that can provide value in the economy. In this chapter the parts about attention residue and distraction resonated the most with me. I'm prone to switch tasks and now that I'm fully remote, it's hard to stay away from Slack and email. I feel like I need to be "seen" by people. And that has made me code slower. 

I do think the equation he gives (High quality work product = (time spent) x (intensity of focus)) is oversimplified. It probably should be logarithmic. Focus should be a sliding scale, the more time you spend the less focus you have.

Finally, there are a couple of points I'm trying to figure out how to incorporate in my own life. That is doing deliberate practice, and being comfortable going deep. I'll tackle the latter point first.

- With the aforementioned distractions, going deep is becoming harder and harder. I'm starting to feel friction or an itch anytime I attempt to do deep work. Going deep is a per-requisite to learn the core abilities. So I hope he touches on the how later, fingers crossed for some strategies!
- Deliberate practice makes a lot of sense if you're an athlete or you have something with a fast feedback loop. But that can be hard to come by as a developer. I think when you're first learning deliberate practice makes total sense. But as you grow as a developer you get to a point where you're making bigger decisions with feedback loops far into the future. I'd love if Newport would show case studies of knowledge workers performing deliberate practice.

**Reply by cmcrawford2** [reply]
My iPhone broke last month (on the very day I got the book and skim read it), so there's one distraction gone. I'm also struggling to not look at Slack and email. I have told myself in the past that I can look at Slack before I start work (as I define work). However yesterday I decided this was not a good idea, because of the residue problem. When I first was parted from my iPhone, I noticed the urge to turn on the radio in the car was strong every time I approached a traffic light (i.e. faced boredom). I just resisted the urge. So it's been about a month and I feel like I'm free of the phone distraction and the urge to do something to replace it, and at the same time I feel like I'm visiting 2021 from a quarter of a century ago. I have been driving without navigation and that took some getting used to even though when I first started driving, there was no navigation app and no cell phones either. Somehow I drove thousands of miles anyway.

What I'm trying to do now is look at Slack at lunch. Because if I were working in an office, I'd be eating lunch with other people. Then at the end of the day.

I spent the weekend in my own Bollingen Tower, which is a cabin by a lake in New Hampshire that we bought last year. Total isolation and beautiful setting. I know I can do a lot better work, but I did work around seven hours a day. What I'm doing now that I call "work" is preparing for an interview with Amazon in a few weeks. I spent a couple of hours making a project plan and then started taking some first steps - set up practice behavioral questions, made a list of technical topics that I need to remember, ran through some coding challenges on Hackerrank. I don't think I'm going to have trouble ramping up the "deep work" for this project because it's important to me. I think I'm what they're looking for but I want to make it easy for them to come to that conclusion.

One other thought is about residues. Since my iPhone broke I've been listening to some CDs from the 2000s. David Allen, the author of the book "Getting Things Done," created a very popular productivity system, and for a while, for a price, he would send out a monthly CD that was an interview with some CEO. I keep things so I still had them. I was reminded of his main thesis, which is that if you have an external system in which you capture your commitments, they won't nag at you. You will have "mind like water," which was his catchphrase. To me this is exactly a method of dealing with residues. If you get an idea, write it down and have a system that you trust to incorporate the idea so that you don't lose it and can deal with it later. Then you won't have the residue. His book Getting Things Done is totally worth a read and I'm so glad rediscovering that system is part of having my phone broken.

I think the idea of having written goals and written plans to meet those goals can substitute for deliberate practice. If you have a goal, and you write down everything you need to do to accomplish it, in that list you will see which things require deep work. In terms of deliberate practice, maybe another idea would be, identify the key areas in which learning new things would advance your value as an employee (or in general as the case may be), and set aside deep work time to learn those new things. This is also classic advice from the productivity guru Brian Tracy.

**Comment by cmcrawford2** [comment]
I'm trying to reconcile my idea of what I want to do with the ideas from chapter 1. I have an interview with Amazon coming up next month and I want to have my biggest project finished. It is the front end for a database of polling results so that users can look at formatted data from a web site, and also the database and back end. But I would also like to talk about my other projects, my React apps and machine learning. I know when I stop working on something for a couple of weeks, I have to backtrack. Plus I study German daily and play the piano daily, and I have significant family and household responsibilities, volunteer commitments and craft projects.
I have a habit of doing German first thing in the morning before breakfast and I don't think I need to drop that. It's something that I picked up for fun last year and it was easy to do daily from the start, actually fun, and I can read German now, which is a skill I don't want to lose. So that stays. Piano stays too. I usually practice just before dinner. I don't think those two things have residues that impinge on work. I can tell my family and the groups I volunteer for that they will need to pick up some slack, but I still have to do a bare minimum. I'm going to batch my craft projects and work on them second half of October.
My biggest problem is the online classes I'm taking. If I don't keep them up then I end up having to go back several sessions. So I guess my plain is going to be to start the day working on the database project and then proceed with the classes in the afternoon and evening. He has a section called "Schedule Every Minute of Your Day" (p. 221 in my hardcover) that I've been doing for a few weeks now.

**Reply by tomcudd** [reply]
It's good to plan. I'd make sure to build some margin into your system for unexpected or unplanned things. Scheduling unplanned work is still scheduling!

**Comment by tomcudd** [comment]
### Sept 15-21: Chapter 2, Deep Work is Rare

- Collaboration, rapid communications, and social media are highly prioritized by many companies. Can this kind of distracted workplace provide any benefit?
- While the book is a few years old, do general trends still skew towards distracted workplaces?
- How difficult is it to measure information work productivity?
- Do you check email/chat outside of "working hours"? Do you "stay perpetually online" because of how easy it is to do?
- Would you work harder if you know that the outcome had a higher chance of benefiting you or have a satisfactory outcome?
- Do you find it challenging to determine what to work on next?
- Are you clear about what matters in your work? Are you aware of what goals you are measured against?
- Do you try to be visibly busy but unproductive because you are worried about the perception of work value?
- Do you feel it necessary to curtail social media and general internet usage in general to maximize deep work?
- Have you seen policies or workplaces that appear to encourage and prioritize deep work?

**Reply by tomcudd** [reply]
A few things I thought of while reading. The idea of an open office, with chat apps on all day, and constant interruptions give me a bit of anxiety now. The extra effort required to start significant work often causes me to redirect to something easier and more comfortable like email or chat. The cult of the internet section seemed to cherry pick only the worst, most disruptive instances of social media and internet. But the book does have to prove a point.

**Reply by cmcrawford2** [reply]
I've done a good job of curtailing social media. Not just because my phone broke. (I got it back today, still haven't unpacked. I think it's dead, but they got my pictures off of it.) Today I looked at email once early in the day and once at the end. Slack only just now. Sadly I forgot a board meeting.
I find it very hard to do deep work at home. Cooking, cleaning, laundry, shopping, appointments are HUGE distractions. Today I had an aye doctor appointment which really destroyed my day. (Routine, eyes are fine).
Anyway I think I'm on the right track with my schedule format and practices. Just today was a challenge.

**Reply by jonathanyeong** [reply]
I thought that he went a bit too hard on the email/rapid communication stuff. I agree, he's trying to prove a point here, but there are plenty of roles that require this type of work. It does feel like he's not giving managers that much credit. Yes their outcomes are ambiguous and hard to measure, but having a good manager makes the world of difference!

Other than that, I do agree open offices are terrible for doing any type of programming/deep work. I'm so much happier working from home. Although I still need to figure out a healthier system than being available on slack all the time. The only policies that I've seen companies do is no meeting days. It's a little bit of a cop out because people that need information from you will either set up a meeting, or you'll have a long slack conversation. I'd love to see more practices that encourage deep work!

**Comment by tomcudd** [comment]
### Sept 22-28: Chapter 3: Deep Work Is Meaningful
- Do you believe deep work can provide satisfaction in your life? Meaning?
- What of these things do you believe are keys to altering or managing how you feel? 
  - "Skillfully managing attention"
  - Your actions
  - What you pay attention to
  - What you focus on
- Can work provide meaning and importance? Do you enjoy work more than free time?
- Do you believe the actual actions of deep work is more important than the content?
- What would a job look like that embraced flow state? Can you discern the meaning of your work?
- Do you understand the medium that you work in? Can you see how your piece of the work you do fits into a larger picture?
- Do you think there's a job that would provide greater satisfaction than tech work?

**Comment by tomcudd** [comment]
### Sept 29-Oct 5: Rule 1, Work Deeply
- Do you believe there is a finite amount of willpower that depletes as you use it?
- Have you considered which deep work philosophy (or combination) will benefit you now?
  - scheduling regular deep work sessions
  - elimination of shallow tasks
  - fully bimodal (alternating time periods of shallow tasks and deep work)
  - journalistic (finding any free moment to quickly dive into deep work mode)
 - Do you believe deep work can happen in small groups of people as well as individually?
 - Would a "calendar" approach crossing off days of completed deep work benefit you?
 - What kind of rituals would you create to help your deep work method?
 - Do you have to make grand gestures to get into a deep work habit?
 - Can you run your life and work like a business?
   - focusing on the important
   - find and act on leading measures
   - keep a scoreboard
   - create accountability structure and cadence
 - Can you shut down email, slack, distractions at a certain time each day?
 - Can you create a ritual that allows you to disconnect and recharge each day?

**Reply by cmcrawford2** [reply]
1. I have a Bollingen tower of my own. We have a cabin in New Hampshire, and my version of shallow work doesn't exist there. That would be laundry, paperwork, cooking for other people and cleaning up after other people. I wouldn't say the work I'm trying to do now is particularly deep - I am trying to learn skills to become employable, but the same strategies apply. Previously I could only go there from Friday to Monday because I had to share the car with my daughter, who needed it for work, but my other daughter moved away and sold me her car, so now I can stay at the cabin until I run out of clean clothes (no washer or dryer there). So I think in reality the bimodal way is best for me. I can try to carve out time for myself at home but the demands of housekeeping and the rest of the family are overwhelming, and I only ever manage a handful of hours a day of focused work.
2. At home I've tried heroically to eliminate the shallow work. Lots of the advice books I've read say to be OK with letting the house go to hell, plants die, etc. But I can't live that way, so dishes have to be washed, laundry put away, plants watered etc. So I've tried to train myself to get those things done on a schedule. If I get them done on time then they don't occupy my mind. And when I'm deliberate about them I can do them faster.
Anyway I still have a lot of room for improvement.

**Comment by tomcudd** [comment]
### Oct 6-12: Rule 2, Embrace Boredom
- How much effort do you need to put in for the process of "gaining knowledge"?
- Do you find that concentration is difficult to come by? Do training efforts seem to improve it?
- Have you tried methods for eliminating digital distractions? A digital detox? Do Not Disturb on your phone? Focus time on calendars?
- Can you queue up things like searches, questions, research on the internet at specific times instead of instantaneously when doing work like software development?
- Can you keep your focus time completely off the internet?
- When waiting in an office, or waiting in line, or sitting in the car, have you tried to embrace that boredom and avoid looking at your phone?
- Have you tried walking, standing, running, biking, commuting, or other travel time methods to focus on problem solutions?
- What skill or ability (like memorization) are you wanting to hone and improve with attention control? Can the methods describe help to do that?

**Reply by cmcrawford2** [reply]
I have totally succeeded in this because my phone broke weeks ago and I have not replaced it yet. I'm not kidding - it took a while to get used to it but now I can just sit and do nothing. I could never do that before. Slack is a problem though. See, I just checked Slack and saw this post. But today I managed to get my friend's database set up on postgres version 12. It was on postgres version 13, but AWS doesn't give you free tier for postgres 13. Also I hadn't imported the 300 survey questions, and that took hours of misery. But now that I can be bored, I could deal with it.

**Comment by tomcudd** [comment]
### Oct 13-19: Rule 3, Quit Social Media
- How do you plan to take back control of your time and attention from the many digital distractions?
- Is there a middle ground of all internet and no internet that provides a maximum benefit and healthy boundary? What does that look like for you?
- Do you use networking tools, websites, apps when they provide you any minimal benefit regardless of their negative impact?
- When it comes to weighing the pros and cons of tech, does the same analytical approach work for every piece of software you install, every website you visit, or every piece of technology you pick up? Do you follow an approach when you install an app, visit a site, or use tech?
- How would you come up with an evaluation rubric for determining what tools you should use?
- Can social media have low or high positive impact on your life at different moments? Like packing away a box of things we don't use or need, when should you get rid of or retain tools like social media?
- Are there services you can drop off now?
- How can you "put more thought" into your leisure time to avoid falling into sites designed to retain your attention for long periods of time?

**Reply by cmcrawford2** [reply]
I've been stressed and that makes me look at social media more than usual. I really have to fight it. The least I can do is write down the hours that I spend on it. I keep thinking I should quit Twitter, but my kid is on Twitter. I guess that is the "any benefit." I stopped following a ton of news outlets on Twitter at the end of 2020 because my reasoning was, it does me no good to know the news, and I don't do anyone any good by knowing it. But news has crept back into my Twitter feed and I'm idly browsing through it almost every day now.

Thinking more about the previous chapter, at the end of it, he talks about concentration exercises, like studying Midrash or memorizing a deck of cards. I have two things that serve that purpose which is that I'm learning to read German and I play the piano. So what I've been trying to do this week and last is to ramp up the intensity with which I do those two things. With piano it's playing longer than I want to. Today I managed an hour. With German it's reading a book that I decided to read rather than just going through my flash card program so that I don't get behind. What I would like to do is get a regular schedule of getting up earlier so that I can spend 30-45 undisturbed minutes on German rather than the 15 minutes it takes to review vocabulary.

**Comment by tomcudd** [comment]
### Oct 20-26: Rule 4, Drain the Shallows
- Does a 4 day work week seem like a good approach to focusing efforts?
- What do you think you could accomplish with your work if you had a month to create and focus on one singular goal?
- Would setting aside and scheduling administrative items to certain blocks of your daily routine help you get into focus the rest of the day?
- Can you be agile with your scheduling and routines to learn how to adjust around interruptions to find focus times?
  - Also consider the use of padding or margin time after tasks that you have a challenging time estimating.
  - Keep in mind that obsessing over the use of a schedule is a form a shallow work that can distract, and try to keep things high level.
  - The goal is to allow your schedule to unlock flow states in your deep work to keep pulling on that useful thread.
- Do you have a shut down routine and specific time you finish your work?
- Is the pressure to be responsive self-imposed?
- Have you trained the people you work with that you will respond promptly to instantaneous forms of communications and set unhealthy expectations?
- Attempting to provide more depth in written communications like slack or email is an solution Cal suggests. Do you see how it can benefit you by reducing the back and forth that happens in these types of channels?

**Reply by tomcudd** [reply]
I love the process centric approach to emails and communications. I despise the lack of clarity that happens in much of our conversations in written form so strive to be as clear as possible. The difference in typing a thoughtful response in 5 minutes instead of shooting over something in 30 seconds often saves me hours.

**Comment by tomcudd** [comment]
### Oct 27-31: Conclusion
Some high level things to think about here in the last chapter.
- Consider that the idea around encouraging deep work is a pragmatic approach to improve the quality of your life and work.
- It is very easy to fall into the shallows, but deep focus on work is a skill that can be re-learned, a muscle that can be strengthened, and new benefits can happen each time you commit towards this kind of effort.
- Some periods in your left can lend itself to an easy entrance into deep work efforts, where other times it takes a commitment to schedule and block off time.
- For many, the goal of deep work is to be able to complete important, valuable tasks for themselves or their organization in an effort to maintain a health balance in life.
- We are striving to be productive and have meaning with our work and lives, and these methods seem like a great approach to meet those goals.

**Comment by cmcrawford2** [comment]
I revived my smartphone today after it was dead for two months, and I just spent an hour and a half with it. I am going to have to figure something out to keep it at bay.
